### PR TITLE
WAR-1588: Case app utils

### PR DESCRIPTION
**Pull request by sanikakulkarni**

### CODE REVIEW

File: katana/wui/core/static/core/js/main.js [hunk 1]
```diff
@@ -1242,4 +1242,67 @@ var katana = {
 
   },
 
+    utils: {
+
+        getRelativeFilepath: function (basePath, path) {
```
Comment (kaheichan) [inline katana/wui/core/static/core/js/main.js]: Trying the following code in chrome dev console
`katana.utils.getRelativeFilepath("/home/ka/Desktop/cases/", "/home/ka/Desktop/cases/warriorframework/katana");
"warriorframework/katana"`
`katana.utils.getRelativeFilepath("/home/ka/Desktop/cases", "/home/ka/Desktop/cases/warriorframework/katana");
"../../../..//home/ka/Desktop/cases/warriorframework/katana"`
The second example isn't expected

File: katana/wui/core/static/core/js/main.js [hunk 2]
```diff
@@ -1242,4 +1242,67 @@ var katana = {
...
+            for (i=hold; i<pathSeries.length; i++) {
+                output += "/" + pathSeries[i]
+            }
+            if (output.startsWith("/")) {
+                output = output.slice(1, output.length);
+            }
+            return output
+        },
+
+        getAbsoluteFilepath: function (basePath, relativePath) {
```
Comment (kaheichan) [inline katana/wui/core/static/core/js/main.js]: Similar situation as the previous comment
`
katana.utils.getAbsoluteFilepath("/home/ka/Desktop/cases/", "warriorframework/katana");
"/home/ka/Desktop/cases/warriorframework/katana"`
`katana.utils.getAbsoluteFilepath("/home/ka/Desktop/cases", "warriorframework/katana");
"/home/ka/Desktop/warriorframework/katana"
`
The second example should append a / at the end if it doesn't have one

File: katana/wui/core/static/core/js/main.js (hunk 1 above)
Comment (sanikakulkarni) [reply katana/wui/core/static/core/js/main.js]: Addressed.

File: katana/wui/core/static/core/js/main.js (hunk 2 above)
Comment (sanikakulkarni) [reply katana/wui/core/static/core/js/main.js]: Addressed.

**Pull request merged**